rct229/rulesets/ashrae9012019/section5/section5rule39.py

```diff
@@ -0,0 +1,50 @@
+from rct229.rulesets.ashrae9012019.data.schema_enums import schema_enums
```

Comment:
I forgot to mention this yesterday. Could you do ```isort (your file name)``` along with black formatting? isort allows to organize import modules.